### ISSUE TITLE
V3 doesn't work with desktop version

### ISSUE THREAD

**Issue by indigolounge** [jamesbechet/tumbTag#3]
I am still on the old version of tumblr. It seems like only the mobile version changed and the updated version doesn't work for me now.  😐

**Comment by indigolounge** [comment]
So I have a workaround if you are still one the old tumblr desktop version, if you reblog by opening a new tab it does work. It's unfortunate but better than nothing. It does not work from the dashboard though.

**Comment by jamesbechet** [comment]
@indigolounge it’s a bit of late reply. I want to confirm, is that issue still occurring?

**Comment by indigolounge** [comment]
It worked again but now it doesn't show up at all. Not sure if Tumblr did a update or if it's something else, but for me it's not working at all.

**Comment by jamesbechet** [comment]
@indigolounge can you please confirm you have the latest version installed?  https://chrome.google.com/webstore/detail/tumblr-tag/cmbakfeanhaniakabiiabhdjkhgbbdmp

<img width="971" alt="Screen Shot 2023-06-21 at 8 46 35 PM" src="https://github.com/jamesbechet/tumbTag/assets/2947763/bd2d3abf-a2c4-4f3b-8e71-42907ff2d2a0">

**Comment by indigolounge** [comment]
Yes that's the one I have.
![image](https://github.com/jamesbechet/tumbTag/assets/63042696/61403678-61d6-435c-a22f-9352f997c4d5)

**Comment by jamesbechet** [comment]
@indigolounge that’s strange.

Can you please open the developer console and type `tumblrtag` and take a screenshot of what you see? That would be very helpful to fix the issue.

<img width="1728" alt="Screen Shot 2023-06-22 at 4 42 32 PM" src="https://github.com/jamesbechet/tumbTag/assets/2947763/7cc691c8-b41e-42a8-90c8-61b4daff7614">

**Comment by indigolounge** [comment]
Hope it helps!

![image](https://github.com/jamesbechet/tumbTag/assets/63042696/73222641-b957-46ab-a280-6d189ac56feb)

**Comment by jamesbechet** [comment]
@indigolounge yes thank you. It was broken on the Text posts. Can you please update it to `3.5.4` and let me kknow if this works for you?

**Comment by indigolounge** [comment]
Works perfectly again! Thank you so much!

**Comment by jamesbechet** [comment]
@indigolounge thank you
